src/extensions/default/JavaScriptCodeHints/node/package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "brackets-javascript-code-hints",
+  "dependencies": {
+    "acorn": "3.3.0",
+    "tern": "0.20.0"
```

Comment:
Latest tern is now `0.21.0`. If we can update to that, great. If it has breaking changes, lets keep this one and update later.